collectors/python.d.plugin/spigotmc/spigotmc.chart.py

```diff
@@ -39,14 +40,25 @@
         'lines': [
             ['users', 'Users', 'absolute', 1, 1]
         ]
+    },
+    'mem': {
+        'options': [None, 'Minecraft Memory Usage', 'mb', 'spigotmc', 'spigotmc.mem', 'line'],
+        'lines': [
+            ['mem_cur', 'Current Memory Usage', 'absolute', 1, 1],
+            ['mem_max', 'Max Memory', 'absolute', 1, 1]
```

Comment:
```suggestion
            ['mem_cur', 'current', 'absolute', 1, 1],
            ['mem_max', 'max', 'absolute', 1, 1]
```

I think we can simplify dimension names, it is clear from the context (The chart title is 'Minecraft Memory Usage') that it is memory.